### PR TITLE
Add information about adding same variant to multiple lines in checkout

### PR DESCRIPTION
**Pull request by SzymJ**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7111 Allow multiple copies of a single variant in the same Checkout](https://app.clickup.com/t/2549495/SALEOR-7111)

File: docs/developer/checkout.mdx [hunk 1]
```diff
@@ -399,50 +533,50 @@ This feature is only available for apps with `HANDLE_CHECKOUTS` permission.
 The custom price can be set with the use of field `price` in the
 [CheckoutLineInput](developer/api-reference/inputs/checkout-line-input.mdx)
 in the following mutations:
+
 - [checkoutCreate](developer/api-reference/mutations/checkout-create.mdx),
 - [checkoutLinesAdd](developer/api-reference/mutations/checkout-lines-add.mdx) - when adding
-a variant that is already added to checkout, the already existing line is overridden - the quantity is incremented and the price is updated.,
+  a variant that is already added to checkout, the already existing line is overridden - the quantity is incremented and the price is updated.,
```
Comment (peelar) [inline docs/developer/checkout.mdx:539]:
```suggestion
  a variant that already exists in the checkout, the corresponding line gets overridden - the quantity is incremented and the price is updated.
```

File: docs/developer/checkout.mdx [hunk 2]
```diff
@@ -496,16 +629,16 @@ If user A adds last two items to their checkout, those two items will become pur
 
 Those two items will also be subtracted from the available quantity displayed to other users on store, giving them immediate feedback that item they are interested in is currently not available, preventing them from adding this item to their own checkouts.
 
-If user A completes their checkout within given time, temporary allocation will be converted into permanent one. But if they don't complete their stock within the time limit, their stock reservation will expire and the two items in their checkout will become available for other users to add to their own checkouts. 
+If user A completes their checkout within given time, temporary allocation will be converted into permanent one. But if they don't complete their stock within the time limit, their stock reservation will expire and the two items in their checkout will become available for other users to add to their own checkouts.
```
Comment (peelar) [inline docs/developer/checkout.mdx:632]:
```suggestion
If user A completes their checkout within the given time, temporary allocation will be converted into a permanent one. But if they don't complete their stock within the time limit, their stock reservation will expire and the two items in their checkout will become available for other users to add to their own checkouts.
```

**Review by peelar — COMMENTED** [review]
Hello 👋 

Two points here:
1. I encourage you to use [Grammarly](https://grammarly.com/). It is a wonderful tool for catching grammar/syntax errors.
2. I noticed you went back and forth on the highlighting of a parameter/variable, so for example one time you wrote `lineId`, and another you wrote **lineId**.

Unless there is a reason for it, I would like us to stick with `lineId`.

Cheers!

**Comment by peelar** [comment]
@SzymJ The build is failing because of broken links. Could you have a look?

**Review by peelar — APPROVED** [review]

**Pull request merged**